unstable test_bgpmon

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
check the log here.

https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-image-pr/7269/testReport/junit/bgp/test_bgpmon/test_bgpmon_no_resolve_via_default/

**Describe the results you received:**

Error Message
Failed: Syn packets is captured when resolve-via-default is disabled
Stacktrace
duthost = <tests.common.devices.MultiAsicSonicHost object at 0x7f326d00af90>
common_setup_teardown = (u'10.1.0.32', '11.0.0.1', [28, 29, 30, 31])
ptfadapter = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>

    def test_bgpmon_no_resolve_via_default(duthost, common_setup_teardown, ptfadapter):
        """
        Verify no syn for BGP is sent when 'ip nht resolve-via-default' is disabled.
        """
        local_addr, peer_addr, peer_ports = common_setup_teardown
        exp_packet = build_syn_pkt(local_addr, peer_addr)
        # Load bgp monitor config
        logger.info("Configured bgpmon and verifying no packet on {} when resolve-via-default is disabled".format(peer_ports))
        try:
            # Disable resolve-via-default
            duthost.command("vtysh -c \"configure terminal\" \
                            -c \"no ip nht resolve-via-default\"")
            duthost.command("sonic-cfggen -j {} -w".format(BGPMON_CONFIG_FILE))
            # Verify no syn packet is received
            pytest_assert(0 == testutils.count_matched_packets_all_ports(test=ptfadapter, exp_packet=exp_packet, ports=peer_ports, timeout=BGP_CONNECT_TIMEOUT),
>                        "Syn packets is captured when resolve-via-default is disabled")
E                        Failed: Syn packets is captured when resolve-via-default is disabled

common_setup_teardown = (u'10.1.0.32', '11.0.0.1', [28, 29, 30, 31])
duthost    = <tests.common.devices.MultiAsicSonicHost object at 0x7f326d00af90>
exp_packet = <ptf.mask.Mask instance at 0x7f326c674370>
local_addr = u'10.1.0.32'
peer_addr  = '11.0.0.1'
peer_ports = [28, 29, 30, 31]
ptfadapter = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>


**Describe the results you expected:**

test pass

**Additional information you deem important:**
<!--
software/ASIC/Hardware Flatform version and info
-->
    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

--- a/tests/bgp/test_bgpmon.py
+++ b/tests/bgp/test_bgpmon.py
@@ -124,6 +124,8 @@ def test_bgpmon(duthost, common_setup_teardown, ptfadapter):
     """
     local_addr, peer_addr, peer_ports = common_setup_teardown
     exp_packet = build_syn_pkt(local_addr, peer_addr)
+    # Flush dataplane
+    ptfadapter.dataplane.flush()
     # Load bgp monitor config
     logger.info("Configured bgpmon and verifying packet on {}".format(peer_ports))
     duthost.command("sonic-cfggen -j {} -w".format(BGPMON_CONFIG_FILE))
@@ -142,6 +144,8 @@ def test_bgpmon_no_resolve_via_default(duthost, common_setup_teardown, ptfadapte
         # Disable resolve-via-default
         duthost.command("vtysh -c \"configure terminal\" \
                         -c \"no ip nht resolve-via-default\"")
+        # Flush dataplane
+        ptfadapter.dataplane.flush()
         duthost.command("sonic-cfggen -j {} -w".format(BGPMON_CONFIG_FILE))
         # Verify no syn packet is received
         pytest_assert(0 == testutils.count_matched_packets_all_ports(test=ptfadapter, exp_packet=exp_packet, ports=peer_ports, timeout=BGP_CONNECT_TIMEOUT),

